### ISSUE TITLE
[Bug]: Hop 2.8.0 - When running a workflow using Hop remote workflow engine, with pipelines containing the "Combination lookup-update" step when building the dispatch zip, the connection properties disappear.

### ISSUE THREAD

**Issue by Raul-rp** [apache/hop#3699]
### Apache Hop version?

2.8.0, 2.7.0

### Java version?

11

### Operating system

Windows

### What happened?

When running a workflow using Hop remote workflow engine, with pipelines containing the "Combination lookup-update" step when building the dispatch zip, the connection properties disappear.

ORIGINAL (Local) :

  <transform>
    <name>Combination lookup/update</name>
    <type>CombinationLookup</type>
    <description/>
    <distribute>Y</distribute>
    <custom_distribution/>
    <copies>1</copies>
    <partitioning>
      <method>none</method>
      <schema_name/>
    </partitioning>
    <cache_size>9999</cache_size>
    <commit>100</commit>
    **<connection>FE0VSQL218</connection>**
    <crc>N</crc>
    <crcfield>hashcode</crcfield>
    <fields>
      <key>
        <lookup>AGENT</lookup>
        <name>AGENT</name>
      </key>
      <return>
        <creation_method>tablemax</creation_method>
        <last_update_field/>
        <name>AGENT_ID</name>
        <use_autoinc>N</use_autoinc>
      </return>
    </fields>
    <preloadCache>N</preloadCache>
    <replace>N</replace>
    <schema>dm_honda_me</schema>
    <table>dim_agent</table>
    <attributes/>
    <GUI>
      <xloc>576</xloc>
      <yloc>224</yloc>
    </GUI>
  </transform>
  
ZIP in Hop server :

  <transform>
    <name>Combination lookup/update</name>
    <type>CombinationLookup</type>
    <description/>
    <distribute>Y</distribute>
    <custom_distribution/>
    <copies>1</copies>
    <partitioning>
      <method>none</method>
      <schema_name/>
    </partitioning>
    <cache_size>0</cache_size>
    <commit>0</commit>
    <crc>N</crc>
    <fields>
      <return>
        <use_autoinc>N</use_autoinc>
      </return>
    </fields>
    <preloadCache>N</preloadCache>
    <replace>N</replace>
    <attributes/>
    <GUI>
      <xloc>576</xloc>
      <yloc>224</yloc>
    </GUI>
  </transform>

### Issue Priority

Priority: 1

### Issue Component

Component: Hop Gui, Component: Hop Server

**Comment by Raul-rp** [comment]
[test.zip](https://github.com/apache/hop/files/14511355/test.zip)
Attached is an example of ETL to be sent to the Hop server that reproduces the bug.

**Comment by Raul-rp** [comment]
Message

**Comment by Raul-rp** [comment]
Same issue in Hop 2.8.0 version

**Comment by RaulBSC** [comment]
Same issue in Hop 2.10.0 version

**Comment by NicoDupont** [comment]
here is the same with the 2.10.0

**Comment by hansva** [comment]
.take-issue

**Comment by NicoDupont** [comment]
The problem seems to persist in 2.11
just testing this morning 


25/01/17 11:56:29 - Recherche dans base de donn�es.0 - ERROR: **Database connection is missing for transform** [Recherche dans base de donn�es]!2025/01/17 11:56:29 - Recherche dans base de donn�es.0 - ERROR: Error initializing transform [Recherche dans base de donn�es]2025/01/17 11:56:29 - Insertion dans table PAIE_ABSENCE_LONGUE_MALADIE.0 - Connected to database [Data Warehouse] (commit=1000)2025/01/17 11:56:29 - calcul_longue_absence - ERROR: Transform [Recherche dans base de donn�es.0] failed to initialize!2025/01/17 11:56:29 - Extraction depuis table DIM_DATE.0 - Finished reading query, closing connection.2025/01/17 11:56:29 - Extraction Absences Type Maladie.0 - Finished reading query, closing connection.2025/01/17 11:56:29 - Extraction depuis table DIM_DATE 2.0 - Finished reading query, closing connection.2025/01/17 11:56:29 - Extraction depuis table FAIT_INDICATEURS_STATS_SOCIALES.0 - Finished reading query, closing connection.HopException: We failed to initialize at least one transform.  Execution can not begin!Root cause: HopException: We failed to initialize at least one transform.  Execution can not begin!org.apache.hop.core.exception.HopException: We failed to initialize at least one transform.  Execution can not begin!
